web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -74,7 +74,8 @@ goog.require('gn_alert');
         },
         'footer':{
           'enabled': true,
-          'showSocialBarInFooter': true
+          'showSocialBarInFooter': true,
+          'showApplicationInfoAndLinksInFooter': true
```

Comment:
Should we hide the footer when it is empty ?
eg. if we turn both options off ? and social links are hidden in record view so when no application info, we just have a grey footer

Reply:
The Pages API allow to add links in the footer.

Reply:
yes I see using
```
  <li gn-static-pages-list-viewer data-section="footer" data-language="{{lang}}" />
```
so maybe not that easy to check if it is completely empty.